docs/static-menu/intro.md

```diff
@@ -13,9 +13,31 @@ The Burendo Handbook is a place where we, as Burendoers, document and standardis
 
 If you want to know how we do something within Burendo, it'll likely be within these pages.  Please use the search functionality to help find what you're looking for, or feel free to browse the pages by category on the left!
 
+## How does the Handbook fit in?
 
-## What this is not
+The Handbook is an important piece of who we are in Burendo and fits in around other tools and resources we have such as the ones below.
 
-This is not the staff handbook, as referred to in your employee contracts.  Please use our internal HR tool, contact your line manager or the People team for assistance.
```

Comment:
Not sure we should get rid of this just yet.  It's our fine print against contractual references to a handbook.

Reply:
Added it back as a note